src/views/portal/VerticalTimeline.tsx

```diff
@@ -0,0 +1,62 @@
+import CheckCircle from "assets/icons/check-circle-fill.svg";
+import XCircle from "assets/icons/x-circle-fill.svg";
+import Image from "next/image";
+import ListGroup from "react-bootstrap/ListGroup";
+import styles from "views/portal/Portal.module.scss";
+
+export interface TimelineProps {
+	status: string | null;
+	submission_time: string | null;
+	verdict_time?: string | null;
+}
+
+function VerticalTimeline(props: TimelineProps) {
```

Comment:
Suggestion: we can destructure individual properties rather than referencing the entire object throughout the component.